packages/v4/patternfly-docs/content/design-guidelines/components/dropdown/dropdown.md

```diff
@@ -3,8 +3,6 @@ id: Dropdown
 section: components
 ---
 
-A **dropdown** presents a menu of actions or links in a constrained space that will trigger a process or navigate to a new location. See [select component guidelines](/components/select/) for more information about making one or more selections from a list of items in a value list.
```

Comment:
For this one, can we link to the design guidelines tab, instead of just the select page, because we're specifically saying "guidelines"